<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageAccountsDialog</class>
 <widget class="QDialog" name="ManageAccountsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>351</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="accountsLabel">
     <property name="text">
      <string>A&amp;ccounts:</string>
     </property>
     <property name="buddy">
      <cstring>tableView</cstring>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsAndTableHorizontalLayout">
     <item>
      <widget class="QTableView" name="tableView">
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="buttonsVerticalLayout">
       <item>
        <widget class="QPushButton" name="addNewAccountButton">
         <property name="text">
          <string>&amp;Add account</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="revokeAuthenticationButton">
         <property name="text">
          <string>&amp;Revoke authentication</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteAccountButton">
         <property name="text">
          <string>&amp;Delete account</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>60</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="statusBarLabel">
     <property name="styleSheet">
      <string notr="true">margin-top: 2px;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
